### PR TITLE
Add super chain.

### PR DESCRIPTION
**Pull request by superexchain**
Add super chain.

### CODE REVIEW

**Comment by yardapen** [comment]
I have me for it guy but someone is trying to steal them do there put up

File: _data/chains/eip155-1970.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "infoURL": "http://superexchain.com",
+  "shortName": "scs",
+  "chainId": 1970,
+  "networkId": 1970,
+  "icon": "super",
+  "explorers": [
+    {
+      "name": "superexscan",
+      "url": "http://superexchain.com",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-1970.json:20]: this does not look like a block explorer & not EIP3091 compatible.

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
you did not address my comment at all

**Comment by superexchain** [comment]
> 4587de5

sorry, i forgot update explorer url

**Pull request merged**